MAUI/MauiAppWithBroker/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,121 @@
+{
+  "Sample": {
+    "Title": "A .NET MAUI MAUI using MSAL.NET and a broker to acquire an access token and call Microsoft Graph API",
+    "Level": 200,
+    "Client": "MAUI (iOS, Android, UWP)",
+    "RepositoryUrl": "active-directory-xamarin-native-v2",
+    "Endpoint": "AAD v2.0",
+    "Description": "A .NET MAUI app using MSAL.NET to sign-in users using a broker and acquiring a token to call Microsoft Graph Api",
+    "Service": "Microsoft Graph",
+    "Languages": [
+      "csharp"
+    ],
+    "Products": [
+      "maui",
+      "azure-active-directory"
+    ],
+    "Library": [
+      "Microsoft Graph"
+    ],
+    "Platform": "MAUI"
+  },
+  "ReadmeAboutTheCode": {
+    "IncludeFilePath": "../ReadmeFiles/AboutTheCode.md"
+  },
+  "ReadmeTroubleshooting": {
+    "IncludeFilePath": "../ReadmeFiles/TroubleShooting.md"
+  },
+  "ReadmeNextSteps": {
+    "IncludeFilePath": "../ReadmeFiles/NextSteps.md"
+  },
+  /*
+    This section describes the Azure AD Applications to configure, and their dependencies
+  */
+  "AADApps": [
+    {
+      "Sample": {
+        "SampleSubPath": "MAUI/MauiAppWithBroker"
+      },
+      "Id": "client",
+      "Name": "active-directory-maui-with-broker-v2",
+      "Kind": "WinUI",
+      "SampleSubPath": "MAUI/MauiAppWithBroker",
+      "AppBrokers": {
+        "HasWebAccountManagerBroker": true,
+        "HasAndroidBroker": true,
+        "HasiOSBroker": true
+      },
+      "RequiredResourcesAccess": [
+        {
+          "Resource": "Microsoft Graph",
+          "DelegatedPermissions": [
+            "User.Read"
+          ]
+        }
+      ]
+    }
+  ],
+  /*
+    This section describes how to update the code in configuration files from the apps coordinates, once the apps
+    are created in Azure AD.
+    Each section describes a configuration file, for one of the apps, it's type (XML, JSon, plain text), its location
+    with respect to the root of the sample, and the mappping (which string in the config file is mapped to which value
+  */
+  "CodeConfiguration": [
+    {
+      "App": "client",
+      "SettingKind": "iOSReplyUrl",
+      "SettingFile": "\\..\\Platforms\\iOS\\AppDelegate.cs",
+      "Mappings": [
+        {
+          "key": "msauth.com.companyname.mauiappwithbroker://auth",
+          "value": "iOSReplyUrl"
+        }
+      ]
+    },
+    {
+      "App": "client",
+      "SettingKind": "AndroidReplyUrl",
+      "SettingFile": "\\..\\Platforms\\Android\\MainActivity.cs",
+      "Mappings": [
+        {
+          "key": "msauth://com.companyname.mauiappwithbroker/ZibsxQEZJWrFF+/959iv+aoQdpU=",
```

Comment:
>ZibsxQEZJWrFF+/959iv+aoQdpU= [](http://example.com/codeflow?start=61&length=28)

Is this some kind of hard coded value?

Reply:
This was from the original application. I can replace this.